appinventor/components/src/com/google/appinventor/components/runtime/Form.java

```diff
@@ -2055,7 +2055,13 @@ public int AccentColor() {
    */
   @DesignerProperty(editorType = PropertyTypeConstants.PROPERTY_TYPE_THEME,
       defaultValue = ComponentConstants.DEFAULT_THEME)
-  @SimpleProperty(userVisible = false, description = "Sets the theme used by the application.",
+  @SimpleProperty(userVisible = false, description = "<p> Pick a design theme for your app. Themes change the appearance of an app, \" +\n" +
+     "  \"such as how buttons and text look. The most common themes are: </p> <ul> <li> <code>Classic</code>: \" + \n" +
+     "  \"This theme stays consistent whether you are looking at an Android, iOS, or the screen layout \" +\n" +
+     "  \"in App Inventor’s designer. Choose Classic if you want detailed control of the appearance \" +\n" +
+     "  \"of your app. </li><li> <code>Device Default</code>: This theme makes your app resemble the other \" +\n" +
+     "  \"apps on your device. With the default theme, however, your app won’t look consistent across \" +\n" +
+     "  \"Android, iOS, and App Inventor’s designer. The best way to see the true appearance of your app is to view it using the Companion.",
```

Comment:
These strings are not right. If you look at them in the help widget you will see it looks like Java code that has been escaped. This should be cleaned up before merge.

Reply:
Forget that. The toolkit description looks all right but the theme does not.